xscen/__init__.py

```diff
@@ -2,28 +2,25 @@
 import warnings
 
 # Import the submodules
-from . import (
-    aggregate,
-    biasadjust,
-    catalog,
-    checkups,
-    common,
-    config,
-    extraction,
-    finalize,
-    indicators,
-    io,
-    regridding,
-    scr_utils,
-)
+from . import aggregate, catalog, checkups, config, indicators, io, scripting, utils
+
+# Import top-level functions
+from ._biasadjust import *
```

Comment:
This would move every function in those modules to the top=level. You sure?

Reply:
Well, with the other changes proposed. I think many functions really don't have to be in `__all__`, since we only use them within the file itself (or sparingly elsewhere) and the end user will never have to call them.